lib/livebook_web/live/hooks/sidebar_hook.ex

```diff
@@ -25,6 +28,23 @@ defmodule LivebookWeb.SidebarHook do
     {:halt, put_flash(socket, :info, "Livebook is shutting down. You can close this page.")}
   end
 
+  defp handle_info(:logout, socket) do
+    {_type, module, _key} = Livebook.Config.identity_provider()
+
+    case module.logout(LivebookWeb.ZTA, socket) do
+      :ok ->
+        Livebook.Users.unsubscribe(socket.assigns.current_user.id)
+
+        {:halt,
+         socket
+         |> assign(current_user: nil)
+         |> redirect(to: ~p"/logout")}
```

Comment:
This is a full redirect, the LV will disappear, so the subscription and assigns will be gone automatically, no?

```suggestion
        {:halt, redirect(socket, to: ~p"/logout")}
```